lib/block-editor-settings.php

```diff
@@ -77,6 +77,8 @@ function gutenberg_get_block_editor_settings( $settings ) {
 	// Copied from get_block_editor_settings() at wordpress-develop/block-editor.php.
 	$settings['__experimentalFeatures'] = gutenberg_get_global_settings();
 
+	$settings['__experimentalFeatures']['layout']['definitions'] = gutenberg_get_layout_definitions();
```

Comment:
I think this still means plugins can edit these (at least in the frontend). So why don't we avoid the setting at all and just hard code them in the client too. Personally I don't mind the duplication (client/server) but potentially if you don't want duplication you can put them in a json file and have a build step or something to generate the php and the JS?

Reply:
Given layout styles are a work in progress (with new layout types such as grid still being worked on), it would be easier for development and less error-prone if these definitions were stored in a single place. A json file might be the best approach if we don't want to add them as block editor settings, though I'm not too concerned about it - the settings being editable by plugins doesn't bother me as much as these definitions being output in generated theme.json files. I assume any plugin author that were to deliberately edit these settings would know and be responsible for what they were doing 😅

Reply:
It turned out not to be too bad copying and pasting to a JS file. Very happy for feedback on it, but since we don't change the definitions _too_ frequently, now that I look at it, I don't think it's too much of an issue if we have to update in two places, so long as it's clear that it has to happen in both places.

I've updated to try out a separate JS `LAYOUT_DEFINITIONS` const.

Reply:
That's perfect for me :)